explorer/lib/explorer_web/live/utils.ex

```diff
@@ -245,9 +262,17 @@ defmodule Utils do
         nil ->
           Logger.debug("Fetching from S3")
 
-          batch.data_pointer
-          |> Utils.fetch_batch_data_pointer()
-          |> Utils.calculate_proof_hashes()
+          batch_content = batch.data_pointer |> Utils.fetch_batch_data_pointer()
+          case batch_content do
+            {:ok, batch_content} ->
+              batch_content
+              |> Utils.calculate_proof_hashes()
+
+            {:error, reason} ->
+              Logger.error("Error fetching batch content: #{inspect(reason)}")
+              # Returning something ensures we avoid attempting to fetch the invalid data again.
+              ["invalid batch"]
```

Comment:
This string is converted to hex in frontend

Reply:
Changed to `[<<0>>]` to show `0x0` in the frontend for now. I'm solving this in a different PR.